src/main/java/org/valdroz/vscript/BaseNode.java

```diff
@@ -353,9 +353,40 @@ public Variant execute(VariantContainer variantContainer) {
                 if ((result == null || result.isNull()) && (valueSubstitution != null)) {
                     result = valueSubstitution.execute(variantContainer);
                 }
+
                 result = Variant.sanitize(result);
             }
             break;
+            case NT_MF_IF: {
+                if (parentRunBlock == null) {
+                    throw new UndefinedFunction(getName());
+                }
+                AbstractFunction function = parentRunBlock.resolveFunction(getName());
```

Comment:
This is not necessary as the node it self is a `if` function

Reply:
purpose of `parentRunBlock` is to resolve function that are not native to vscript engine

Reply:
updated the `NT_MF_IF` block.